tests/test262-es6-excludelist.xml

```diff
@@ -1,5 +1,32 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <excludeList>
+  <reason>These tests should throw SyntaxError instead of ReferenceError based on the ES11 standard</reason>
+  <test id="language/types/boolean/S8.3_A2.1.js"></test>
+  <test id="language/types/boolean/S8.3_A2.2.js"></test>
+  <test id="language/types/reference/S8.7.2_A1_T1.js"></test>
+  <test id="language/types/reference/S8.7.2_A1_T2.js"></test>
+  <test id="language/reserved-words/S7.6.1_A1.1.js"></test>
+  <test id="language/reserved-words/S7.6.1_A1.2.js"></test>
+  <test id="language/reserved-words/S7.6.1_A1.3.js"></test>
+  <test id="language/expressions/this/S11.1.1_A1.js"></test>
+  <test id="language/keywords/S7.6.1.1_A1.18.js"></test>
+  <test id="language/expressions/prefix-increment/non-simple.js"></test>
+  <test id="language/expressions/prefix-decrement/non-simple.js"></test>
+  <test id="language/expressions/postfix-increment/non-simple.js"></test>
+  <test id="language/expressions/postfix-decrement/non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/add-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/btws-and-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/btws-or-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/btws-xor-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/div-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/left-shift-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/mod-div-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/mult-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/right-shift-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/subtract-non-simple.js"></test>
+  <test id="language/expressions/compound-assignment/u-right-shift-non-simple.js"></test>
+  <test id="language/expressions/assignment/non-simple-target.js"></test>
```

Comment:
This list is ordered, please insert these into the right place. Reasons should be after them (can be shorter: `ES11 change: Should throw Syntaxerror`).

Reply:
if you use the `--test262-es2015 update` command it, places the failed tests at the end of the file. That simplifies our job since the tests at the end of the file are skipped due to some reason.

Reply:
I asked Ádám to place these tests to a separated block. It makes easier to maintain the excludelist. We will have to blocks: tests won't be fixed ever and tests should be fixed.

Just a nit: Reason tag is inside the test tag normally. But we don't use junit to process test results, so a simple HTML comment is good for us to notice why we won't fix these test failures.